eZ/Publish/API/Repository/Exceptions/ContentFieldValidationException.php

```diff
@@ -13,7 +13,7 @@
 /**
  * This Exception is thrown on create or update content one or more given fields are not valid.
  */
-abstract class ContentFieldValidationException extends ForbiddenException
+abstract class ContentFieldValidationException extends ContentValidationException
```

Comment:
given ContentValidationException is no longer deprecated, but we just adjust when ContentFieldValidationException over ContentValidationException, then I think we can safely remove this change.

By removing this change we won't have to introduce bc breaks like you showed below now, where ContentFieldValidationException will match catch blocks for ContentValidationException meaning you1'll need to adjust order of them.

For this PR that will mean simplifying diff in `eZ/Publish/Core/REST/Server/Controller/Content.php`.